### PR TITLE
clean up unused macros in the makefile

### PR DESCRIPTION
**Pull request by jxchinaren**

### CODE REVIEW

File: buildtools.py [hunk 1]
```diff
@@ -173,15 +173,7 @@ def buildMakefile(CTX):
     for dir in CTX.OBJ_INCLUDE_DIRS:
         MAKECPPFLAGS += " -I${OBJDIR}/%s" % (dir)
     MAKECPPFLAGS += " -I${OBJDIR}"
-    # I don't think these are used anywhere.
```
Comment (BillWhite) [inline buildtools.py]: I left these in, but they can probably be taken out.

File: buildtools.py [hunk 2]
```diff
@@ -362,8 +352,10 @@ def buildMakefile(CTX):
                       replaceExtension(jni_objname, ".d")]
         makefile.write("\n")
     makefile.write("\n")
+
```
Comment (BillWhite) [inline buildtools.py]: This is a slightly different comment.  It says "For debugging" rather than "for debugging the makefile".

**Comment by BillWhite** [comment]
Seems plausible to me.

**Comment by jxchinaren** [comment]
Jenkins passed and will go and merge it now.

**Pull request merged**